utils/utils.go

```diff
@@ -0,0 +1,27 @@
+package utils
+
+import (
+	"os"
+	"strconv"
+	"strings"
+)
+
+// Retrieves the specified environmental variable with type of Boolean as string, set it with defualt variable if not exist or unacceptable format
+func getBoolAsStringEnvVar(env, defaultVal string) string {
```

Comment:
this function should just return a boolean. No need to cast it back to a string after we already have it in boolean format